cdk/lib/mparticle-api.ts

```diff
@@ -26,6 +28,44 @@ export class MParticleApi extends GuStack {
 			},
 		});
 
+		// API Gateway
+		new GuApiGatewayWithLambdaByPath(this, {
+			app: app,
+			targets: [
+				{
+					path: '/data-subject-requests/{requestId}/callback',
+					httpMethod: 'POST',
+					lambda: lambda,
+				},
+			],
+			// Create an alarm
+			monitoringConfiguration: {
+				snsTopicName: `alarms-handler-topic-${this.stage}`,
+				http5xxAlarm: {
+					tolerated5xxPercentage: 1,
+				},
+			},
+		});
+
+		// SQS Queue
+		const mparticleCallbackDeadLetterQueue = new Queue(
+			this,
+			'MparticleApiCallbackDeadLetterQueue',
+			{
+				queueName: `mparticle-api-callback-dead-letter-queue-${this.stage}`,
+				retentionPeriod: Duration.seconds(864000),
```

Comment:
is this a thing?  If not 24 * 60 * 60 would make more sense
```suggestion
				retentionPeriod: Duration.hours(24),
```

Reply:
24h doesn't sound long if we have to investigate, especially if something happens on the weekend